### PR TITLE
Force to use us-east-1 for our acceptance tests

### PR DESCRIPTION
**Pull request by halprin**
### Information
Fixes our acceptance tests in master.

### Changes proposed in this PR:
- Forces us to use `us-east-1` with our acceptance tests since CircleCI doesn't know what region to use.  We were already forcing the region with other clients we were making in our acceptance tests, so now we are consistent.

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [x] New unit tests written to cover new functionality.
- [x] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [x] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

**Comment by halprin** [comment]
Good catch, @omnivion.  I'll fix this up shortly.

**Comment by halprin** [comment]
Fixed, @omnivion!

File: test-commons/src/main/java/gov/cms/qpp/test/helper/AwsTestHelper.java [hunk 1]
```diff
@@ -19,11 +21,25 @@
  * A helper class for aws testing
  */
 public class AwsTestHelper {
-	private static AmazonDynamoDB dynamoClient = AmazonDynamoDBClientBuilder.standard().build();
-	private static AmazonS3 s3Client = AmazonS3ClientBuilder.standard().withRegion(Regions.US_EAST_1).build();
 	public static final String TEST_DYNAMO_TABLE_NAME = "qpp-qrda3converter-acceptance-test";
+
+	private static AmazonDynamoDB dynamoClient = AmazonDynamoDBClientBuilder.standard().withRegion(Regions.US_EAST_1).build();
```
Comment (omnivion) [inline test-commons/src/main/java/gov/cms/qpp/test/helper/AwsTestHelper.java]: why not also final

Comment (halprin) [reply test-commons/src/main/java/gov/cms/qpp/test/helper/AwsTestHelper.java]: Oof, yes, you are correct.  Fixing...  😝

**Pull request merged**